src/tainting/Taint.ml

```diff
@@ -130,6 +130,16 @@ and compare_sources s1 s2 =
       Option.compare compare_precondition s1.precondition s2.precondition
   | other -> other
 
+and compare_args a1 a2 =
+  let pos1 = a1.pos in
+  let pos2 = a2.pos in
+  match Stdlib.compare pos1 pos2 with
+  | 0 ->
+      List.compare
+        (fun n1 n2 -> IL_helpers.compare_name n1 n2)
```

Comment:
In this particular case I think `IL_helpers.compare_name` is preferable. Not need to change, just nit picking.

Reply:
Thanks for pointing it out, I wrote this when I had multiple conditions and forgot to change it back